docs/rules/no-continue.md

```diff
@@ -0,0 +1,65 @@
+# Disallow continue (no-continue)
+
+This rule prevents the use of `continue` statement:
```

Comment:
Need more of an explanation here. This section should be an introduction to `continue` and what it does, and then explain why some people don't like using it.